src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,890 @@
+    "\n",
+    "        calculate_variable_from_constraint(\n",
+    "            unit.cond_duty[t],\n",
+    "            unit.eq_cond[t])\n",
+    "\n",
+    "        calculate_variable_from_constraint(\n",
+    "            unit.retentate.flow_mol_phase_comp[t, \"Liq\", \"water\"],\n",
+    "            unit.eq_retentate_flow[t, \"Liq\", \"water\"])\n",
+    "\n",
+    "        calculate_variable_from_constraint(\n",
+    "            unit.retentate.flow_mol_phase_comp[t, \"Liq\", \"ethylene_glycol\"],\n",
+    "            unit.eq_retentate_flow[t, \"Liq\", \"ethylene_glycol\"])\n",
+    "\n",
+    "        calculate_variable_from_constraint(\n",
+    "            unit.retentate.temperature[t],\n",
+    "            unit.temperature_out_calculation[t])\n",
+    "\n",
+    "        calculate_variable_from_constraint(\n",
+    "            unit.retentate.pressure[t],\n",
+    "            unit.pressure_out_calculation[t])\n",
+    "\n",
+    "    assert degrees_of_freedom(unit) == 0\n",
+    "    if degrees_of_freedom(unit) == 0:\n",
+    "        res = solver.solve(unit, tee = True)\n",
+    "    unit.inlet.flow_mol_phase_comp.unfix()\n",
+    "    unit.inlet.temperature.unfix()\n",
+    "    unit.inlet.pressure.unfix()\n",
+    "    print('Custom initialization routine complete: ', res.solver.message)\n",
+    "\n",
+    "solver = get_solver()\n",
+    "m.fs.M101.initialize()\n",
+    "propagate_state(m.fs.s03)\n",
+    "m.fs.pervap.config.initializer = my_initialize\n",
+    "my_initialize(m.fs.pervap)\n",
+    "\n",
+    "results = solver.solve(m, tee = True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's check the results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.M101                                                             Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                                  Units       water_feed  glycol_feed   Outlet  \n",
+      "    Molar Flowrate ('Liq', 'water')            mole / second     0.34000  1.0000e-06     0.34000\n",
+      "    Molar Flowrate ('Liq', 'ethylene_glycol')  mole / second  1.0000e-06     0.66000     0.66000\n",
+      "    Temperature                                       kelvin      318.15      318.15      318.15\n",
+      "    Pressure                                          pascal  1.0132e+05  1.0132e+05  1.0132e+05\n",
+      "====================================================================================\n",
+      "Question for reviewers: Is there a way to add report() method with a stream table?\n",
+      "\n",
```

Comment:
Presently, it doesn't look like there's a way to do this, but it would be good to add, and (probably) not that hard to implement.

Obviously remove this before we merge.

Reply:
As per @radhakrishnatg 's suggestion, I will just remove the comment for now. We can update this example if the `SkeletonModel` gains a `report()` method in the future.